fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -271,6 +271,8 @@ public Response createOrReplaceObjectRdf(
 
         if (httpConfiguration.putRequiresIfMatch() && StringUtils.isBlank(ifMatch) && !resource.isNew()) {
             throw new ClientErrorException("An If-Match header is required", 428);
+        } else if (resource.hasType(FEDORA_PAIRTREE)) {
+            throw new ClientErrorException("Graphs cannot be created for resources under pairtree nodes", FORBIDDEN);
```

Comment:
I would please propose the following in response to this:

* ForbiddenResourceModificationException
* UnsupportedResourceModificationException
* IllegalResourceModification
* BadResourceModification

My preference lies primarily with the term "Forbidden" (the mapping to the HTTP 403 response description feels appropriate), but I could certainly defend "Unsupported" as well.

"Illegal" and "Bad" concern me given that they may easily be applicable within situations in which, I would speculate, a 405 response is appropriate.  "Cannot" also concerns me in this regard (as well as it being identified as undesirable).